x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/find_comments.ts

```diff
@@ -81,36 +86,77 @@ export default ({ getService }: FtrProviderContext): void => {
       expect(caseComments.comments).to.eql(patchedCase.comments);
     });
 
-    it('should filter case comments', async () => {
+    it('should find only case comments of the correct type', async () => {
       const { body: postedCase } = await supertest
         .post(CASES_URL)
         .set('kbn-xsrf', 'true')
         .send(postCaseReq)
         .expect(200);
 
-      // post 2 comments
+      // post 5 comments of all possible types
       await supertest
-        .post(`${CASES_URL}/${postedCase.id}/comments`)
+        .post(INTERNAL_BULK_CREATE_ATTACHMENTS_URL.replace('{case_id}', postedCase.id))
```

Comment:
Let's use the helper function here: https://github.com/elastic/kibana/blob/main/x-pack/test/cases_api_integration/common/lib/api/attachments.ts#L82